src/main/java/org/springframework/samples/petclinic/owner/Owner.java

```diff
@@ -95,9 +95,17 @@ public List<Pet> getPets() {
 
 	public void addPet(Pet pet) {
```

Comment:
Because this function will actually add or update the pet, consider renaming it.